src/garage/torch/algos/ppo.py

```diff
@@ -75,19 +80,24 @@ def __init__(self,
                          policy_ent_coeff=policy_ent_coeff,
                          use_softplus_entropy=use_softplus_entropy,
                          stop_entropy_gradient=stop_entropy_gradient,
-                         entropy_method=entropy_method)
+                         entropy_method=entropy_method,
+                         minibatch_size=minibatch_size,
+                         max_optimization_epochs=max_optimization_epochs)
 
         self._lr_clip_range = lr_clip_range
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
-        """Compute objective using surrogate value and clipped surrogate value.
+    def _compute_objective(self, advantages, obs, actions, rewards):
+        """Compute objective value.
```

Comment:
all docstrings using a `:math:` environment must be prefaced with `r` so that they can use backticks

e.g. 

```python
r"""
Args:
    foo (torch.Tensor): Foos with shape :math:`(F, )`
"""
```

Reply:
Thanks!